script/release

```diff
@@ -159,6 +164,11 @@ main() {
     add_changed_files $major $minor $patch
     commit $major $minor $patch
     push $major $minor $patch
+
+    # publish artifacts
+    script/publish
```

Comment:
I'm not sure if we want to publish artifacts here since we haven't merged the release PR at this point

Reply:
Hmm yeah I see what you mean. The `push` function prints a message asking you to merge the PR, then waits for you to press a key. Is that enough or should we keep the steps separated?

Reply:
I think it is safer to keep them separated

Reply:
Ok I'm 🆒 with that 😎